content/applications/websites/livechat.rst

```diff
@@ -109,28 +130,31 @@ The *Livechat Window* is the space where the live chat conversation with website
 place.
 
 Edit the :guilabel:`Welcome Message` to change the message a visitor sees when they open a new chat
-session. This message will appear as though it is sent by a live chat operator, and acts as both a
+session. This message appears as though it is sent by a live chat operator, and acts as both a
 greeting and an invitation to continue the conversation.
 
 Edit the :guilabel:`Chat Input Placeholder` to alter the text that appears in the box where visitors
-will type their replies.
+type their replies. This message prompts the visitor to initiate the chat.
 
 The *Channel Header* is the colored bar at the top of the chat window. The :guilabel:`Channel Header
-Color` can be changed following the same steps as the *Livechat Button Color* above.
+Color` can be changed following the same steps as the :ref:`Livechat button
+<livechat/livechat-button>`.
 
 .. figure:: livechat/chat-window.png
    :align: center
 
    The Livechat Window with a purple header. The chat input placeholder reads "Ask Something..."
 
+.. _livechat/channel-rules-tab:
+
 Channel rules
 -------------
 
-The :guilabel:`Channel Rules` tab on the live chat channel details form determines when the *Live
-Chat Window* opens on the website, by configuring when a :guilabel:`URL Regex` action is triggered
-(e.g., a page visit).
+The :guilabel:`Channel Rules` tab on the live chat channel detail form determines when the live chat
+window opens on the website, by configuring when a :guilabel:`URL Regex` action is triggered, such
+as a page visit.
```

Comment:
I was slightly confused by the sentence, so how about something like:

To configure which website user actions open the live chat window, go to the ...